admin/app/AppLoader.scala

```diff
@@ -43,14 +46,12 @@ trait AdminServices {
   lazy val rebuildIndexJob = wire[RebuildIndexJob]
 }
 
-trait Controllers extends AdminControllers {
-  def wsClient: WSClient
+trait AppComponents extends FrontendComponents with AdminControllers with AdminServices with I18nComponents {
+
+  lazy val messages: Messages = Messages(langs.preferred(langs.availables), messagesApi)
```

Comment:
I'm not sure about how it's defined in Play 2.4, but in Play 2.5 I'm fairly sure you can direcly use the `MessageApi` like that

```scala
class MyController(messageApi) extends Controller {
  val myMessage: String = messageApi("blah")
}
```

So I don't think you need to instantiate `Message` at all